deployment/common/changeset/link_transfer_timelock.go

```diff
@@ -0,0 +1,102 @@
+package changeset
+
+import (
+	"fmt"
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/mcms"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/timelock"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/shared/generated/link_token"
+)
+
+type LinkTransfer struct {
+	To    common.Address
+	Value *big.Int
+}
+type LinkTransferTimelockRequest struct {
+	Transfers       map[uint64][]LinkTransfer
+	ValidUntil      uint32        // unix time until the proposal will be valid
+	MinDelay        time.Duration // delay for timelock worker to execute the transfers.
+	OverrideRoot    bool
+	StartingOpCount map[uint64]uint64
+}
```

Comment:
Dont we need a `validate` method here for the request object? I see this deployment/common/changeset/transfer_link_token.go , so maybe we should follow it too?

Reply:
added validation https://github.com/smartcontractkit/chainlink/pull/15512/commits/c3e528c8ad744a1b9f789ee73320d4a9e9a3d805